src/Execute.jl

```diff
@@ -19,5 +21,37 @@ function evaluate_node(c::Chromosome, node::OutputNode, context::Vector{BitStrin
 end
 
 function execute_chromosome(c::Chromosome, context::Vector{BitString})
+    c.active_set = true
     return BitString[evaluate_node(c, node, context) for node = c.outputs]
 end
+
+# Supplies the standard context for up to 4 inputs
```

Comment:
See the attached notes document.  I think I have a longer version, but I
don't have time to find it now. And I will write up a longer, more
detailed version.

Note the transposed format for the truth table where columns correspond
to rows in a conventional truth table.  We want each output of the
chromosome to be the bit string that represents one of the rows of the
transposed truth table.

For 3 inputs, we want the first input to be 11110000 binary of 0xF0, the
second input to be 11001100 = 0xCC, and the third input to be 10101010 =
0xAA.  So the standard context for 3 inputs would be these three inputs.
 For 4 inputs, they are  0xFF00,  0xF0F0, 0xCCCC, 0xAAAA, and for 2
inputs  0xC, 0xA.

On Thu, Feb 5, 2015, at 18:17, George Lesica wrote:
> >      return BitString[evaluate_node(c, node, context) for node = c.outputs]
> >  end
> > +
> > +# Supplies the standard context for up to 4 inputs
> 
> What is a standard context?
> 
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/glesica/CGP.jl/pull/2/files#r24216301